scripts/lua/management/lib/swagger.lua

```diff
@@ -98,23 +98,24 @@ end
 function parseRateLimit(swagger, policies)
   local rlObj = swagger["x-gateway-rate-limit"]
   rlObj = (rlObj == nil) and swagger["x-ibm-rate-limit"] or rlObj
+  local unit
```

Comment:
Previously, it was modifying the swagger file directly, resulting in the modified swagger being stored and being returned to the user. By making this a local variable, it keeps the original swagger with `minute`, `second`, etc. instead of it being modified to `60`, `1`, etc.